src/components/PostsList.tsx

```diff
@@ -1,85 +1,94 @@
+  closeCommentList: React.Dispatch<React.SetStateAction<boolean>>,
+  isCommentListHiden: boolean,
+};
 
-    <table className="table is-fullwidth is-striped is-hoverable is-narrow">
-      <thead>
-        <tr className="has-background-link-light">
-          <th>#</th>
-          <th>Title</th>
-          <th> </th>
-        </tr>
-      </thead>
+export const PostsList: React.FC<Props> = ({
+  posts,
+  selectPost,
+  openerDetails,
+  closeCommentList,
+  isCommentListHiden,
+}) => {
+  if (!posts.length) {
+    return (
+      <div
+        className="notification is-warning"
+        data-cy="NoPostsYet"
+      >
+        No posts yet
+      </div>
+    );
+  }
 
-      <tbody>
-        <tr data-cy="Post">
-          <td data-cy="PostId">17</td>
+  return (
+    <div data-cy="PostsList">
+      <p className="title">Posts:</p>
 
-          <td data-cy="PostTitle">
-            fugit voluptas sed molestias voluptatem provident
-          </td>
+      <table className="table is-fullwidth is-striped is-hoverable is-narrow">
+        <thead>
+          <tr className="has-background-link-light">
+            <th>#</th>
+            <th>Title</th>
+            <th> </th>
+          </tr>
+        </thead>
 
-          <td className="has-text-right is-vcentered">
-            <button
-              type="button"
-              data-cy="PostButton"
-              className="button is-link is-light"
-            >
-              Open
-            </button>
-          </td>
-        </tr>
+        <tbody>
+          {posts.map(post => (
+            <tr data-cy="Post" key={post.id}>
+              <td data-cy="PostId">{post.id}</td>
 
-        <tr data-cy="Post">
-          <td data-cy="PostId">18</td>
+              <td data-cy="PostTitle">
+                {post.title}
+              </td>
 
-          <td data-cy="PostTitle">
-            voluptate et itaque vero tempora molestiae
-          </td>
+              <td className="has-text-right is-vcentered">
+                {isCommentListHiden && (
+                  <button
+                    type="button"
+                    data-cy="PostButton"
+                    className="button is-link is-light"
+                    onClick={() => openerDetails(post)}
+                  >
+                    Open
+                  </button>
+                )}
 
-          <td className="has-text-right is-vcentered">
-            <button
-              type="button"
-              data-cy="PostButton"
-              className="button is-link"
-            >
-              Close
-            </button>
-          </td>
-        </tr>
+                {!isCommentListHiden && (
+                  post.id === selectPost?.id
+                    ? (
+                      <button
+                        type="button"
+                        data-cy="PostButton"
+                        className="button is-link is-ligth"
+                        onClick={() => closeCommentList(true)}
```

Comment:
There is a typo in the className property. It should be 'is-light' instead of 'is-ligth'.